src/test/rustdoc/strip-block-doc-comments-stars.rs

```diff
@@ -0,0 +1,11 @@
+#![crate_name = "foo"]
+
+// The goal of this test is to answer that it won't be generated as a list because
+// block doc comments can have their lines starting with a star.
+
+// @has foo/fn.foo.html
+// @snapshot docblock - '//*[@class="rustdoc-toggle top-doc"]//*[@class="docblock"]'
+/**
+ *     a
```

Comment:
I think this is actually supposed to be interpreted as a code block though?

Reply:
Based on the previous behaviour, it's not apparently (I was surprised too).

Reply:
Oh right... I got mixed up. It was incorrectly treated as a code block because the `*` was interpreted as Markdown text. So it's correct to not treat it as a code block.